fe/fe-core/src/main/java/com/starrocks/catalog/Table.java

```diff
@@ -272,6 +276,10 @@ public String getTableIdentifier() {
         return name;
     }
 
+    public String getDbName() {
```

Comment:
I would suggest make it `Optional<String> mayGetDbName()`, bc some types of table may never support it

Reply:
So just throw exception if not suported.

Remember there a lot of places that call `getDbName()` directly.